tests/FSharp.Compiler.ComponentTests/Language/MultiDimensionalArrayTests.fs

```diff
@@ -0,0 +1,45 @@
+﻿namespace FSharp.Compiler.ComponentTests.Language
+
+open Xunit
+open FSharp.Test.Compiler
+
+module MultiDimensionalArrayTests =
+
+    [<Theory>]
+    [<InlineData(1,"array2d")>]
```

Comment:
I think there should be spaces after the commas. Maybe just run `fantomas` on the file.

Reply:
Fantomas is disabled for /tests folder for all invocations.
Reformatted with ignore files turned off, should be OK now.